tools/bazel-compilation-database.sh

```diff
@@ -4,7 +4,7 @@
 # - https://github.com/grailbio/bazel-compilation-database
 # - https://stackoverflow.com/questions/61015990/how-do-i-enable-c-intellisense-for-a-bazel-project-in-vs-code
 
-INSTALL_DIR="$HOME/.local/bin"
+INSTALL_DIR="/usr/local/bin"
```

Comment:
Just so we don't change this for everyone else too. No need to run this more than once yourself either so should all be good.
```suggestion
INSTALL_DIR="$HOME/.local/bin"
```